modules/FormModel.js

```diff
@@ -47,56 +47,58 @@
+     *     custom push method     // a callback that will be invoked when pushing data to the Object model
+     *   }
+     * },
+     * models: {
+     *   modelName: modelInstance,  // optionally, provide a set of model instance to model name (aliases) to start tracking
+     *   modelName2: modelInstance2 // provide as many aliases to model instances as you'd like
      * }
      * @method constructor
      * @param [options] {Object}
-     *   @param [options.models] {Array} list of model configurations. These will dictate what fields from the Object model will be
-     *     used during the pulling and pushing. Will be ignored if options.model exists.
-     *   @param [options.model] {Backbone.Model} An object model to track. Short hand for options.models of size one.
-     *   @param [options.fields] {Array} An array of fields to track off of options.model. If left not defined, all fields will
-     *     be tracked.
-     *   @param [options.computed] {Array} list of computed value configurations. These give you flexibility in how fields are copied
-     *     to and from any number of Object models.
+     *   @param [options.mapping] {Object} map from aliases (either model names or computed value names) to mappings.
+     *     A model mapping can bind an alias to a space seperated list of fields to track as a String  r the boolean true if it is mapping all the
+     *     fields. A computed mapping can bind an alias to a set of model mappings required for this computed value and both a pull and/or push method
+     *     that are used to compute different values to or from object model(s).
+     *   @param [options.models] {Object} Because the options.mapping parameter only allows you to define the mappings to aliases, this options allows
+     *     you to bind model instances to aliases. Setting model instances to aliases are required to actually begin pulling/pushing values.
      *   @param [options.startUpdating=false] {Boolean} set to true if you want to immediately set up listeners to update this form
      *     model as the object model updates. You can always toggle this state with startUpdating() and stopUpdating().
      *   @param [options.validation] {Object} A Backbone.Validation plugin hash to dictate the validation rules
      *   @param [options.labels] {Object} A Backbone.Validation plugin hash to dictate the attribute labels
      */
     constructor: function(attributes, options) {
-      NestedModel.apply(this, arguments);
       options = options || {};
-      this.__computed = [];
       this.__cache = {};
       this.__currentUpdateEvents = [];
-      this.__modelConfigs = [];
-      this.__initMappings(options);
+      this.__currentMappings = {};
+      this.__currentObjectModels = {};
 
       // override + extend the validation and labels hashes
       this.validation = _.extend({}, this.validation || {}, options.validation || {});
```

Comment:
do you need the || {}?  Doesn't _.extend already take care of null/undefined, etc?